packages/@react-spectrum/tag/src/TagGroup.tsx

```diff
@@ -12,19 +12,24 @@
 
 import {ActionButton} from '@react-spectrum/button';
 import {AriaTagGroupProps, TagKeyboardDelegate, useTagGroup} from '@react-aria/tag';
-import {classNames, useDOMRef, useStyleProps} from '@react-spectrum/utils';
+import {classNames, SlotProvider, useDOMRef, useStyleProps} from '@react-spectrum/utils';
 import {DOMRef, StyleProps} from '@react-types/shared';
 import {FocusScope} from '@react-aria/focus';
 import {ListCollection} from '@react-stately/list';
 import {mergeProps, useLayoutEffect, useResizeObserver, useValueEffect} from '@react-aria/utils';
-import React, {ReactElement, useCallback, useMemo, useState} from 'react';
+import React, {ReactElement, ReactNode, useCallback, useMemo, useState} from 'react';
 import styles from '@adobe/spectrum-css-temp/components/tags/vars.css';
 import {Tag} from './Tag';
 import {useLocale} from '@react-aria/i18n';
 import {useProviderProps} from '@react-spectrum/provider';
 import {useTagGroupState} from '@react-stately/tag';
 
-export interface SpectrumTagGroupProps<T> extends AriaTagGroupProps<T>, StyleProps {}
+export interface SpectrumTagGroupProps<T> extends AriaTagGroupProps<T>, StyleProps {
+  /**
+   * Provide one or more ActionButtons to be rendered next to the tag group.
+   */
+  actions?: ReactNode
```

Comment:
```suggestion
  actions?: ReactElement<SpectrumActionButtonProps> | ReactElement<ReactFragment & {children: ReactElement<SpectrumActionButtonProps>[]}>
```
Should we restrict our types to ActionButtons?

Reply:
Good idea, I think so

Reply:
I don't think TS actually does anything with that. Children aren't really type checked.